php/util.php

```diff
@@ -15,10 +15,11 @@
 	// Suppress include warnings if the user disables do_diagnostic
 	// For compatibility with 3rd party plugins which use autoloaders
 	global $do_diagnostic;
-	if($do_diagnostic)
-		include_once 'utility/'. strtolower($class). '.php';
-	else
-		@include_once 'utility/'. strtolower($class). '.php';
+	$file = __DIR__.'/utility/'. strtolower($class). '.php';
```

Comment:
We can't rely on `__DIR__`. If php is not configured correctly, ruTorrent will not work.